### PR TITLE
Add zookeeper tickscripts

### PR DESCRIPTION
**Pull request by jackzampolin**
Have alerts for amount of data held and API latency

### CODE REVIEW

File: examples/telegraf/zookeeper/zookeeper_data_size_batch.tick [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+// TELEGRAF CONFIGURATION
+// [inputs.zookeeper]
+//   servers = [":2181"]
+
+// DEFINE: kapacitor define zookeeper_data_size_batch -type batch -tick zookeeper/zookeeper_data_size_batch.tick -dbrp telegraf.autogen
+// ENABLE: kapacitor enable zookeeper_data_size_batch
+
+// Parameters
+var info = 100000000 // 100MB
```
Comment (nathanielc) [inline examples/telegraf/zookeeper/zookeeper_data_size_batch.tick]: We should make this easier to use. Possibly by defining a sizeUnit var and then multiplying the thresholds by it. For example

```
var mb = 1024*1024 // 1 MB
var gb = 1024*mb // 1 GB

var info = 100 * mb 
var warn = 1 * gb 
var crit = 10 * gb
```

It demonstrates some of the flexibility of TICKscript and it reads better to me. Thoughts?

Comment (jackzampolin) [inline examples/telegraf/zookeeper/zookeeper_data_size_batch.tick]: +100. Thats a WAY more readable format.

File: examples/telegraf/zookeeper/zookeeper_data_size_batch.tick [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+// Dataframe
+var data = batch
+  |query('''SELECT mean(approximate_data_size) AS stat FROM "telegraf"."autogen"."zookeeper"''')
+    .period(period)
+    .every(every)
+    .groupBy('host')
+        
+// Thresholds
+var alert = data
+  |eval(lambda: sigma("stat"))
```
Comment (nathanielc) [inline examples/telegraf/zookeeper/zookeeper_data_size_batch.tick]: An alternative approach would be to dived the stat here by `mb` or `gb` and then set the thresholds in those units.

```
var info = 0.1 // 100 MB
var warn = 1 // 1 GB
var crit = 10 // 1 GB
```

```
|eval(lambda:"stat" / gb, lambda: sigma("stat"))
   .as('stat', 'sigma')
|...
```

Comment (jackzampolin) [inline examples/telegraf/zookeeper/zookeeper_data_size_batch.tick]: I went with the other option

**Comment by nathanielc** [comment]
@jackzampolin Thanks!

**Pull request merged**